### ISSUE TITLE
[BUG] Colab installs are unstable

### ISSUE THREAD

**Issue by benjaminpkane** [voxel51/fiftyone#772]
TLDR; Colab support is unstable until this issue can be resolved

When working with Colab notebooks today, importing `fiftyone` suddenly stopped working in fresh runtimes. This issue also arose for me when installing versions `fiftyone` that previously worked, e.g. `fiftyone==0.7.0.1` and `fiftyone-db==0.2.0`, so I would assume something has changed about Colab itself.

At a minimum, the issue relates to the main process timing out when requesting the port number of the database service. This issue was resolved during development of Colab support, but now it seems to have reappeared for different reasons.

Traceback:
```
---------------------------------------------------------------------------
ServiceListenTimeout                      Traceback (most recent call last)
<ipython-input-3-34017a3769a1> in <module>()
----> 1 import fiftyone

12 frames
/usr/local/lib/python3.6/dist-packages/fiftyone/__init__.py in <module>()
     22 __path__ = extend_path(__path__, __name__)
     23 
---> 24 from fiftyone.__public__ import *
     25 import fiftyone.constants as _foc
     26 from fiftyone.utils.uid import _get_user_id

/usr/local/lib/python3.6/dist-packages/fiftyone/__public__.py in <module>()
      9 import fiftyone.core.service as fos
     10 
---> 11 _database_service = fos.DatabaseService()
     12 config = foc.load_config()
     13 

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in __init__(self)
    196 
    197     def __init__(self):
--> 198         super().__init__()
    199 
    200     @property

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in __init__(self)
     77         self.child = None
     78         if not self._disabled:
---> 79             self.start()
     80 
     81     def __del__(self):

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in start(self)
    281         import fiftyone.core.odm.database as food
    282 
--> 283         food.set_default_port(self.port)
    284         food.get_db_conn()
    285 

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in port(self)
    268     @property
    269     def port(self):
--> 270         return self._wait_for_child_port()
    271 
    272     def start(self):

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in _wait_for_child_port(self, port, timeout)
    175             raise ServiceListenTimeout(etau.get_class_name(self), port)
    176 
--> 177         return find_port()
    178 
    179     @classmethod

/usr/local/lib/python3.6/dist-packages/retrying.py in wrapped_f(*args, **kw)
     47             @six.wraps(f)
     48             def wrapped_f(*args, **kw):
---> 49                 return Retrying(*dargs, **dkw).call(f, *args, **kw)
     50 
     51             return wrapped_f

/usr/local/lib/python3.6/dist-packages/retrying.py in call(self, fn, *args, **kwargs)
    210                 if not self._wrap_exception and attempt.has_exception:
    211                     # get() on an attempt with an exception should cause it to be raised, but raise just in case
--> 212                     raise attempt.get()
    213                 else:
    214                     raise RetryError(attempt)

/usr/local/lib/python3.6/dist-packages/retrying.py in get(self, wrap_exception)
    245                 raise RetryError(self)
    246             else:
--> 247                 six.reraise(self.value[0], self.value[1], self.value[2])
    248         else:
    249             return self.value

/usr/local/lib/python3.6/dist-packages/six.py in reraise(tp, value, tb)
    701             if value.__traceback__ is not tb:
    702                 raise value.with_traceback(tb)
--> 703             raise value
    704         finally:
    705             value = None

/usr/local/lib/python3.6/dist-packages/retrying.py in call(self, fn, *args, **kwargs)
    198         while True:
    199             try:
--> 200                 attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
    201             except:
    202                 tb = sys.exc_info()

/usr/local/lib/python3.6/dist-packages/fiftyone/core/service.py in find_port()
    173                 except psutil.Error:
    174                     pass
--> 175             raise ServiceListenTimeout(etau.get_class_name(self), port)
    176 
    177         return find_port()

ServiceListenTimeout: fiftyone.core.service.DatabaseService failed to bind to port
```

**Comment by benjaminpkane** [comment]
Continuing to monitor this. It is almost certainly related to as of yet inexplicable changes in Colab VMs. As of this writing installs are working again.

**Comment by benjaminpkane** [comment]
Stale
